src/search-form/index.jsx

```diff
@@ -0,0 +1,32 @@
+import './style.css'
+
+import {useState} from 'react'
+
+import Button from '../common/button'
+import Input from '../common/input'
+
+const SearchForm = () => {

```

Comment:
according to the task AC, this component should have 2 properties. here there is none